core/src/test/resources/input.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<catalog>
+   <cd>
+      <title>Empire Burlesque</title>
+      <artist>Bob Dylan</artist>
```

Comment:
If I haven't said this already, I really appreciate your musical tastes! 💯

Reply:
Well, it comes from https://www.w3schools.com/xml/xsl_transformation.asp :D

Reply:
Another one of my dreams is shattered! 🎼